alf/environments/igibson/suite_igibson.py

```diff
@@ -0,0 +1,384 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import gym
+import numpy as np
+import pybullet as p
+
+from alf.environments import suite_gym
+from alf.environments.gym_wrappers import BaseObservationWrapper, ImageChannelFirst, FrameGrayScale
+import alf
+
+from igibson.objects.visual_marker import VisualMarker
+from alf.environments.igibson.igibson_tasks import VisualPointNavFixedTask, \
+    VisualPointNavRandomTask, VisualObjectNavTask
+from alf.environments.igibson.igibson_env import iGibsonCustomEnv
+
+
+@alf.configurable()
+class FlattenWrapper(BaseObservationWrapper):
```

Comment:
Probably we can directly use built-in gym.wrappers to do this (just provide all keys we are interested):

``` python
        try:  # for modern Gym (>=0.15.4)
            from gym.wrappers import FilterObservation, FlattenObservation
            env = FlattenObservation(FilterObservation(env, keys))
        except ImportError:  # for older gym (<=0.15.3)
            from gym.wrappers import FlattenDictWrapper  # pytype:disable=import-error
            env = FlattenDictWrapper(env, keys)
```

so no need to define our own wrapper.